### ISSUE TITLE
[CI] UpgradeClusterClientYamlTestSuiteIT.test {p0=upgraded_cluster/10_basic/Verify date_nanos default format}

### ISSUE THREAD

**Issue by andreidan** [elastic/elasticsearch#70577]
**Build scan**:
https://gradle-enterprise.elastic.co/s/o5wvhzrsbfmqu

**Repro line**:

**Reproduces locally?**:

**Applicable branches**: `7.x`

**Failure history**:

**Failure excerpt**:
```
"stash" : {	
    "body" : {	
      "error" : {	
        "root_cause" : [	
          {	
            "type" : "index_not_found_exception",	
            "reason" : "no such index [index_nanos]",	
            "resource.type" : "index_or_alias",	
            "resource.id" : "index_nanos",	
            "index_uuid" : "_na_",	
            "index" : "index_nanos",	
            "stack_trace" : "[index_nanos] IndexNotFoundException[no such index [index_nanos]]	
	at org.elasticsearch.cluster.metadata.IndexNameExpressionResolver$WildcardExpressionResolver.indexNotFoundException(IndexNameExpressionResolver.java:986)	
	at org.elasticsearch.cluster.metadata.IndexNameExpressionResolver$WildcardExpressionResolver.innerResolve(IndexNameExpressionResolver.java:923)	
	at org.elasticsearch.cluster.metadata.IndexNameExpressionResolver$WildcardExpressionResolver.resolve(IndexNameExpressionResolver.java:879)	
	at org.elasticsearch.cluster.metadata.IndexNameExpressionResolver.concreteIndices(IndexNameExpressionResolver.java:219)	
	at org.elasticsearch.cluster.metadata.IndexNameExpressionResolver.concreteIndices(IndexNameExpressionResolver.java:193)	
	at org.elasticsearch.action.search.TransportSearchAction.resolveLocalIndices(TransportSearchAction.java:590)
```

**Comment by cbuescher** [comment]
Reproduces for me on 7.x with 
```
./gradlew ':qa:rolling-upgrade:v6.8.15#upgradedClusterTest' -Dtests.class="org.elasticsearch.upgrades.UpgradeClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=upgraded_cluster/10_basic/Verify date_nanos default format}" -Dtests.seed=BCABB78B35998163 -Dtests.security.manager=true -Dtests.bwc=true -Dtests.locale=es-PA -Dtests.timezone=Europe/Kaliningrad -Druntime.java=8 -Dtests.rest.suite=upgraded_cluster
```

**Comment by cbuescher** [comment]
This will be solved when #70855 is backported to 7.x
